autosubmit/autosubmit.py

```diff
@@ -891,35 +893,35 @@ def _init_logs(args, console_level='INFO', log_level='DEBUG', expid='None'):
             if args.command == "stop":
                 exp_id = "_".join(expids)
                 Log.set_file(os.path.join(BasicConfig.GLOBAL_LOG_DIR,
-                                          args.command + exp_id + '.log'), "out", log_level)
+                                          args.command + exp_id + '.log'), "out", args.compress_logs, log_level)
```

Comment:
Is it not the third argument the log level?

```suggestion
                                          args.command + exp_id + '.log'), "out", log_level, args.compress_logs)
```

The same happened with all the changes of the `Log.set_file` function in this file.